pset-functions.js

```diff
@@ -208,6 +315,26 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function doubleLetters(word) {
+  // Have to split the sentence into words
+  // first get rid of possible commas --
+  const characters=word.split("")
+  let double=false
+  //loop through the number of words in the sentence and
+  //check for each of the words we have already identifies
+  for (var i = 0; i < (word.length)-1; i++) {
+    if (characters[i]===characters[i+1]) {
+      double=true
+    }
+  }  //closing the todoloop
+ return double
+} // closing the function
+
+const testword="runu"
+console.log("Test: " + testword)
+console.log("Result for double character test:", doubleLetters(testword))
+
+
```

Comment:
@AtleLohrmann Good work here on this here. Your looping through each letter and comparing it against the next letter, good job coming to that solution. One suggestion since your using the boolean to determine if the consecutive letter exist you could `return true` on line 327 and `return false` on line 330. the benefit of that is if the letter repeats itself twice when your `return true` that stops the program and doesn't continue searching the other letters since you got your answer.
take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)